NorthstarLauncher/main.cpp

```diff
@@ -419,19 +419,14 @@ int main(int argc, char* argv[])
 	{
 		PrependPath();
 
-		if (!fs::exists("ns_startup_args.txt"))
+		// NOTE [Fifty]: Functionality removed in https://github.com/R2Northstar/NorthstarLauncher/pull/517
+		if (fs::exists("ns_startup_args.txt"))
 		{
-			std::ofstream file("ns_startup_args.txt");
-			std::string defaultArgs = "-multiple";
```

Comment:
This PR means we lose the functionality of these default args right?

If so, this could cause issues with the game complaining about multiple instances if an instance is stuck in the background or something